headers/types/common/common.h

```diff
@@ -36,6 +53,47 @@ struct file_stream {
 };
 ASSERT_SIZE(struct file_stream, 72);
 
+// Struct used to load, store and unload WTE files
```

Comment:
I'm not really sure what a WTE file is. Could you add some cursory explanation to this comment about what these files are?

Reply:
I'd be delighted!